docs/reference/upgrade.asciidoc

```diff
@@ -16,13 +16,15 @@ endif::[]
 {es} can usually be upgraded using a <<rolling-upgrades,Rolling upgrade>>
 process so upgrading does not interrupt service. Rolling upgrades are supported:
 
-* Between minor versions
+// tag::rolling-upgrade-versions[]
+* Between minor versions of the same major version
```

Comment:
@debadair You mind taking a look at this. It's a pretty minor change, but it's important for clarity.